pkg/ratelimit/limitware/middleware.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/google/exposure-notifications-verification-server/pkg/controller"
 	"github.com/google/exposure-notifications-verification-server/pkg/database"
 	"github.com/google/exposure-notifications-verification-server/pkg/observability"
+	"github.com/opencensus-integrations/redigo/redis"
```

Comment:
nit: this should be in the last package group of imports

Reply:
Done